### PR TITLE
New try to minimaps improvements

### PR DESCRIPTION
**Pull request by pandaro**
Edit:
the DevPaks27.zip is now in my dropbox, all works fine, but, please feel free to bring back DevPaks.zip in the old places

### CODE REVIEW

File: Sources/Client/ScoreboardView.cpp [hunk 1]
```diff
@@ -241,6 +257,50 @@ namespace spades {
...
+						{128,255,255},//24 light azure
+						{255,255,128},//25 light yellow
+						{255,128,255},//26 light pink2
+						
+						{165,42,42},//27 brown
+						{255,69,0},//28 orange	red
+						{255,165,0},//29  orange
+						
+						{139,69,19},//30 maroon medium
+						{210,105,30},//31 choccolate
```
Comment (theunknownxy) [inline Sources/Client/ScoreboardView.cpp]: This table is exactly the same as in MapView.cpp. I think this should be deduplicated.

Comment (pandaro) [inline Sources/Client/ScoreboardView.cpp]: yes i know, but i dont know  whether it is better position to put it in: MapView.cpp or ScoreboardView.cpp.
Second question. This requires something in particular about the place, or the syntax, where it needs to be declared the color palette?

Comment (pandaro) [inline Sources/Client/ScoreboardView.cpp]: think it is appropriate to define the table within the namespace MapView.h openspades?
something like this:
http://pastebin.com/xhFB0H1C

Comment (pandaro) [inline Sources/Client/ScoreboardView.cpp]: Ok now there is only one definition of  the array palette.
other issues?

**Comment by pandaro** [comment]
No one have other issues?

**Pull request merged**